common.yaml

```diff
@@ -0,0 +1,47 @@
+openapi: 3.1.0
+info:
+  title: Common components
+  version: 1.0.0
+
+components:
+  responses:
```

Comment:
This file will be removed once the branch I copied it from is merged and this branch is rebased.